### PR TITLE
chore(readme): updated readme with styles prop

### PR DESCRIPTION
**Pull request by Budaa**
## Description
Updated readme with new `style` and added props spread.
### Checklist for this pull request
- [x] PR is scoped to one task
- [ ] Tests included
- One of:
  - [x] The PR is not making any UI change
  - [ ] The PR is making a UI change but not a product change
    - [ ] Screenshots included
 
Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,84 @@
 # React Native Zapp Bridge
 
-A React Native package that include bridging from React Native to [Applicaster](http://www.applicaster.com) Zapp Platform native framework.
+A React Native package that include bridging from React Native to
```
Comment (lomudi) [inline README.md]: why separating to two lines?

File: README.md [hunk 2]
```diff
@@ -1,68 +1,84 @@
...
 
 React-Native-Zapp-Bridge is a util package that enable you to:
 
-1. **Analytics** - use `sendAnalyticEvent` to send an analytics event throw [Zapp Morpheus](http://developer-zapp.applicaster.com/analytics/morpheus/morpheus.html) (Zapp Analytics Manager). 
+1. **Analytics** - use `sendAnalyticEvent` to send an analytics event throw
+   [Zapp Morpheus](http://developer-zapp.applicaster.com/analytics/morpheus/morpheus.html)
+   (Zapp Analytics Manager).
 
-2. **Video Player** - use `APVideoPlayer` to add a video player to your React-Native screen, this will add the chosen pluggable player on the Zapp Platform.
+2. **Video Player** - use `APVideoPlayer` to add a video player to your
```
Comment (lomudi) [inline README.md]: why separating lines?

File: README.md [hunk 3]
```diff
@@ -1,68 +1,84 @@
...
+
+1. import:
+
+```javascript
+import { sendAnalyticEvent } from 'react-native-zapp-bridge';
+```
+
+2. send event:
+
+```javascript
```
Comment (lomudi) [inline README.md]: whats the change in the sendAnalyticsEvent?

**Review by lomudi — COMMENTED** [review]
few md questions, the player part looks good. @Budaa

File: README.md (hunk 3 above)
Comment (Budaa) [reply README.md]: I have a code formatting on save enabled...

**Review by lomudi — DISMISSED** [review]
💯

**Review by lomudi — DISMISSED** [review]
💯

**Review by lomudi — APPROVED** [review]
💯

**Pull request merged**